packages/lesswrong/components/tagging/FilterMode.tsx

```diff
@@ -126,15 +149,23 @@ const FilterModeRawComponent = ({tagId="", label, mode, canRemove=false, onChang
     // We just found it, it's guaranteed to be in the defaultVisibilityTags list
     mode = defaultVisibilityTags.get().find(t => t.tagId === tagId)!.filterMode
   }
-  const reducedName = userHasNewTagSubscriptions(currentUser) ? 'Reduced' : "0.5x"
-  const reducedVal = userHasNewTagSubscriptions(currentUser) ? 'Reduced' : 0.5
+  
+  const reducedName = 'Reduced'
+  const reducedVal = 'Reduced'
+  const filterMode = filterModeToStr(mode, currentUser)
+  const isLW = forumTypeSetting.get() === 'LessWrong'
+  const filterModeLabel = isLW
+    ? filterModeStrToLWStr(filterMode)
+    : filterMode
```

Comment:
With these changes, the setting isn't super-readable on the EA Forum because it blends in with the tag name somewhat:
![Screenshot 2022-10-10 at 12 38 47](https://user-images.githubusercontent.com/5075628/194858121-91d9df9e-33ab-471c-8cdd-e6db2e5ee3b4.png)

Probably we need to do one of:

- Make some CSS changes here
- Use `filterModeStrToLWStr` on the EA Forum too
- Make a separate `filterModeStrToEAStr` function if we want something different

Tagging @jpaddison3 for (potentially strong) design opinions

Reply:
I was told this would be forum-gated